lua/entities/lunasflightschool_missile.lua

```diff
@@ -188,8 +188,15 @@ if SERVER then
 				effectdata:SetNormal( -self:GetForward() )
 			util.Effect( "manhacksparks", effectdata, true, true )
 
+			local dmgDone = 600

```

Comment:
nit: This variable name implies the damage has already been applied. I'd prefer something like `damageAmount` or similar

Reply:
Applied!